src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -430,6 +431,34 @@ export default class QueryEditorUI extends Component<Props, State> {
       },
       provideCompletionItems: this.provideCompletionItems,
       prepend: this.props.prepend,
+      footerItems: {
+        start: [
+          <EuiText size="xs" color="subdued" className="queryEditor__footerItem">
+            {`${this.state.lineCount ?? 1} ${
+              this.state.lineCount === 1 || !this.state.lineCount ? 'line' : 'lines'
+            }`}
+          </EuiText>,
+          <EuiText size="xs" color="subdued" className="queryEditor__footerItem">
+            {this.props.query.dataset?.timeFieldName || ''}
+          </EuiText>,
+          <QueryResult queryStatus={this.props.queryStatus!} />,
+        ],
+        end: [
+          <EuiButtonEmpty
+            iconSide="left"
+            iconType="clock"
+            iconGap="s"
+            size="xs"
+            onClick={this.toggleRecentQueries}
+            className="queryEditor__footerItem"
+            flush="both"
+          >
+            <EuiText size="xs" color="subdued">
```

Comment:
is there a reason we're styling the text over the button itself?

Reply:
Button's sizes are not in sync with text size; an `xs` button uses `s` text and there isn't a size below it. Additionally, button doesn't have a `subdued` color.

Reply:
I guess my question is why we are creating designs that oui doesn't support, but that is not blocking. If this is to try to match mocks, that's fine.

Reply:
It is to match mocks for now